### PR TITLE
Remove image license files

### LINKED ISSUE #236: image_LICENSE.txt needs updating
Once PR #233 is merged, there will be no more images in `apptools`.  The `image_LICENSE.txt` file should be updated

### PR DESCRIPTION
**Pull request by aaronayres35**
fixes #236 

The issue was motivated by this comment: https://github.com/enthought/apptools/pull/233#discussion_r529301544

Note, in making this PR I did noticed there is still one images directory in `apptools`: https://github.com/enthought/apptools/tree/master/apptools/logger/plugin/view/images
but it has its own `image_LICENSE.txt` file: https://github.com/enthought/apptools/blob/master/apptools/logger/plugin/view/images/image_LICENSE.txt

This PR also removes the deleted files from the `MANIFEST.in` file.  

**Checklist**
- [ ] Add a news fragment if this PR is news-worthy for end users. (see docs/releases/README.rst)

### CODE REVIEW

**Comment by rahulporuri** [comment]
I'd actually recommend moving the internal image license file - https://github.com/enthought/apptools/blob/master/apptools/logger/plugin/view/images/image_LICENSE.txt - to the top level, and including that in the manifest file. Also, I don't know what `image_LICENSE_CP.txt` is and I don't see that in the repository. That will also need to be added, moved to the top level (root git directory) and included in the manifest.

File: image_LICENSE.txt [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-Project                 License                   File
-----------------------------------------------------------------------------
-Enthought               BSD-like                  LICENSE.txt
-Nuvola                  LGPL                      image_LICENSE_Nuvola.txt
-OOo                     LGPL                      image_LICENSE_OOo.txt
+  Crystal Project: LGPL license as described in image_LICENSE_CP.txt
 
-Unless stated in this file, icons are work of Enthought, and are released
+Unless stated in this file, icons are work of enthought, and are released
```
Comment (rahulporuri) [inline image_LICENSE.txt:6]:
```suggestion
Unless stated in this file, icons are work of Enthought, and are released
```

File: image_LICENSE.txt [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 under BSD-like license.
+
+Files and orginal authors:
```
Comment (rahulporuri) [inline image_LICENSE.txt:9]:
```suggestion
Files and original authors:
```

**Review by rahulporuri — APPROVED** [review]
Mostly LGTM

**Comment by rahulporuri** [comment]
@aaronayres35 i'm not sure where you got the `image_LICENSE_CP.txt` file.

**Comment by aaronayres35** [comment]
> @aaronayres35 i'm not sure where you got the `image_LICENSE_CP.txt` file.

@rahulporuri I found it in `envisage` and assumed it was supposed to be the same file (that assumption should definitely be confirmed though!)

**Pull request merged**